ethereum/contracts/dev-contracts/test/VerifierTest.sol

```diff
@@ -53,6 +53,63 @@ contract VerifierTest is Verifier {
 
             mstore(VK_LOOKUP_TABLE_TYPE_X_SLOT, 0x12cd873a6f18a4a590a846d9ebf61565197edf457efd26bc408eb61b72f37b59)
             mstore(VK_LOOKUP_TABLE_TYPE_Y_SLOT, 0x19890cbdac892682e7a5910ca6c238c082130e1c71f33d0c9c901153377770d1)
+
+            mstore(VK_RECURSIVE_FLAG, 0)
+        }
+    }
+}
+
+/// @author Matter Labs
+contract VerifierRecursiveTest is Verifier {
```

Comment:
Should be separate file per each contract

Reply:
Should I also separate unit_tests?

Reply:
You can write separate contract, js test can be the same